README.md

```diff
@@ -12,11 +12,16 @@ The scrapers are kept separate from Zenodotus, because the IP addresses of tradi
 1. Create `config/application.yml` and create `INSTAGRAM_USER_NAME`, `INSTAGRAM_PASSWORD`.
 1. Run `$ rails secret` and then add a variable `secret_key_base` to `config/application.yml`
 1. `$ rails db:migrate` to setup the database (this uses SQLite so there's no need for Postgres or MySQL or anything)
+1. Setup Selenium standalone server
+	1. Download the "Selenium Server (Grid)" JAR package at https://www.selenium.dev/downloads/
+	1. Save it to the folder of this package
```

Comment:
@cguess Following these instructions causes Git to list it as a new file in the staging area. Can you add a `.gitignore` rule? Presumably for any `.jar` but I leave that up to your wisdom.

Reply:
Ah damn. I did that for the gem but forgot here. Will do.